.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,97 @@
+name: Pre-release tests
+
+on:
+  pull_request:
```

Comment:
Are you suggesting we run the pre-release tests on every PR?  One problem of a pre-merge approach is that the use of [traditional COPR repos](https://copr.fedorainfracloud.org/coprs/g/avocado/avocado-latest/), for instance, would not be possible, a deployment based on the packit COPR repos would be necessary.  If that's the way to go, how can we wait for the packit build, capture its generated repository information, and inject into a workflow here?

Reply:
> Are you suggesting we run the pre-release tests on every PR? 

No, I was using that while debugging the pipeline and forgot to update it. I was planning to run this after a manual trigger (workflow_dispatch) and weekly.